katello.gemspec

```diff
@@ -39,6 +39,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "stomp"
   gem.add_dependency "scoped_search", ">= 4.1.9"
 
+  gem.add_dependency "qpid_messaging"
```

Comment:
This is going to break on EL8 where we can't build this gem as an RPM currently. I think we should drop this requirement here and where you require the gem, do so conditionally if its present and otherwise don't and disable the functionality.